Makefile

```diff
@@ -0,0 +1,15 @@
+LOG_LEVEL ?= info
+MODULES_TOML ?= examples/modules.toml
+
+.PHONY: build
+build:
+	cargo build --release
+
+.PHONY: run
+run:
+	RUST_LOG=$(LOG_LEVEL) cargo run --release -- -c $(MODULES_TOML)
```

Comment:
Nit: Since this is an example, consider dropping the `--release` flag as most people won't have the build artifacts cached for a release, but for normal debug

Reply:
The reason for this is that we run with `--release` on in dev because it makes a HUGE different in the performance of wasmtime (like, with swift it cuts 30 seconds off the execution of `hello world`). I could rename the target if you think that would be better, but I think that both @radu-matei and I run it this way all the time, and it's the documented way to run it according to the README.